OpenRA.Mods.Common/Traits/Render/DrawLineToTarget.cs

```diff
@@ -139,7 +139,7 @@ public static void ShowTargetLines(this Actor self)
 		{
 			// Target lines are only automatically shown for the owning player
 			// Spectators and allies must use the force-display modifier
-			if (self.Owner != self.World.LocalPlayer)
+			if (self.World.IsGameOver && !self.World.LocalPlayer.CanControlUnitsOf(self.Owner))
```

Comment:
Why the extra `IsGameOver` condition here?

Reply:
Otherwise Red Alert crashes in main menu with the battle scene in background. I don't see this happening with Dune 2000 or Tiberian Sun.

```
Loading mod: ra
Exception of type `System.NullReferenceException`: Object reference not set to an instance of an object.
   at OpenRA.Mods.Common.Traits.LineTargetExts.ShowTargetLines(Actor self) in OpenRa/OpenRA.Mods.Common/Traits/Render/DrawLineToTarget.cs:line 142
   at OpenRA.Mods.Common.Traits.Mobile.OpenRA.Traits.IResolveOrder.ResolveOrder(Actor self, Order order) in OpenRa/OpenRA.Mods.Common/Traits/Mobile.cs:line 939
   at OpenRA.Mods.Common.Traits.Wanders.DoAction(Actor self, CPos targetCell) in OpenRa/OpenRA.Mods.Common/Traits/Wanders.cs:line 121
   at OpenRA.Mods.Common.Traits.Wanders.TickIdle(Actor self) in OpenRa/OpenRA.Mods.Common/Traits/Wanders.cs:line 84
   at OpenRA.Actor.Tick() in OpenRa/OpenRA.Game/Actor.cs:line 277
   at OpenRA.World.Tick() in OpenRa/OpenRA.Game/World.cs:line 428
   at OpenRA.Game.InnerLogicTick(OrderManager orderManager) in OpenRa/OpenRA.Game/Game.cs:line 623
   at OpenRA.Game.LogicTick() in OpenRa/OpenRA.Game/Game.cs:line 638
   at OpenRA.Game.Loop() in OpenRa/OpenRA.Game/Game.cs:line 803
   at OpenRA.Game.Run() in OpenRa/OpenRA.Game/Game.cs:line 856
   at OpenRA.Game.InitializeAndRun(String[] args) in OpenRa/OpenRA.Game/Game.cs:line 294
   at OpenRA.Launcher.Program.Main(String[] args) in OpenRa/OpenRA.Launcher/Program.cs:line 32
```

Reply:
because `self.World.LocalPlayer` is null

Reply:
I guess [48a1110](https://github.com/OpenRA/OpenRA/pull/19144/commits/48a11104a9cda4f74b46852006c3d10a361491de) is a better solution.